d2.config.js

```diff
@@ -1,11 +1,11 @@
 const config = {
     type: 'app',
-    name: 'app-management',
+    name: 'dhis-web-app-management',
```

Comment:
`app-management` is the correct value for `name` - the full name with `dhis-web-` is the URL used by the backend

Reply:
```suggestion
    name: 'app-management',
```

Reply:
Thanks 😁